launcher/Application.cpp

```diff
@@ -673,8 +676,32 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
 
         m_settings->registerSetting("UpdateDialogGeometry", "");
 
-        // pastebin URL
-        m_settings->registerSetting("PastebinURL", "https://0x0.st");
+        // HACK: This code feels so stupid is there a less stupid way of doing this?
+        {
+            m_settings->registerSetting("PastebinURL", "");
+            m_settings->registerSetting("PastebinType", PasteUpload::PasteType::Mclogs);
+            m_settings->registerSetting("PastebinCustomAPIBase", "");
+
+            QString pastebinURL = m_settings->get("PastebinURL").toString();
+
+            bool userHadNoPastebin = pastebinURL == "";
+            bool userHadDefaultPastebin = pastebinURL == "https://0x0.st";
+            if (!(userHadNoPastebin || userHadDefaultPastebin))
```

Comment:
nit: IMO this is easier to read

```suggestion
            bool userHadDefaultPastebin = pastebinURL == "https://0x0.st";
            if (!pastebinURL.isEmpty() && !userHadDefaultPastebin)
```

Reply:
I agree.

Reply:
I'll change it to this when I go on my laptop

Reply:
Actually I can just commit it directly